recipes/derekstride-tree-sitter-sql/all/CMakeLists.txt

```diff
@@ -0,0 +1,50 @@
+cmake_minimum_required(VERSION 3.15)
+project(tree-sitter-sql LANGUAGES C)
+
+find_package(tree-sitter REQUIRED CONFIG)
+include(GenerateExportHeader)
+
+file(WRITE api.h [[
```

Comment:
I want to understand only 1 thing about this. Is this file then not provided by upstream either? Or what is its pourpose here?

Reply:
@AbrilRBS 
I just follow the recipe of `tree-sitter-c`.
https://github.com/conan-io/conan-center-index/blob/master/recipes/tree-sitter-c/all/CMakeLists.txt

I think all recipes of tree-sitter in cci should have same style as much as possible.

Reply:
@AbrilRBS 
I just follow the recipe of `tree-sitter-c`.
https://github.com/conan-io/conan-center-index/blob/master/recipes/tree-sitter-c/all/CMakeLists.txt

I think all recipes of tree-sitter in cci should have same style as much as possible.